src/modules/shared/components/Navbar/Navbar.tsx

```diff
@@ -1,7 +1,74 @@
-import React from 'react'
+import { useContext, useState } from 'react';
+import { HiMiniBellAlert } from 'react-icons/hi2';
+import { RiArrowDropDownLine } from 'react-icons/ri';
+import { Link, useNavigate } from 'react-router-dom';
+import human from '../../../../assets/human.png';
+import logo from '../../../../assets/logo.png';
+import { AuthContext } from '../../../../context/AuthContext/AuthContext';
 
 export default function Navbar() {
+  const { loginData } = useContext(AuthContext);
+  const navigate = useNavigate();
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+
+  const handleLogout = () => {
+    localStorage.clear()// Clear token
+    navigate('/login'); // Redirect to login page
+  };
+
   return (
-    <div>Navbar</div>
-  )
+    <>
+      <div className="flex flex-col md:flex-row gap-5 md:gap-0 justify-between items-center bg-gray-50 shadow-lg py-1  px-5">
+        <div onClick={() => navigate('/dashboard')} className="cursor-pointer">
```

Comment:
It should be a link with the `to` prop to the dashboard, as the only thing it does is navigation.